smartseq2_single_sample/check_reference/reference_check_wf.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
included

Reply:
I'm aware it's in the bucket, but I more meant that having to go to a bucket to see what the inputs to your wdl were was onerous, particularly because they weren't typed in the file, but rather abstracted by the Array[Object] type.